components/eventing-controller/controllers/subscription/jetstream/reconciler.go

```diff
@@ -20,8 +20,9 @@ import (
 	"github.com/kyma-project/kyma/components/eventing-controller/controllers/events"
 	"github.com/kyma-project/kyma/components/eventing-controller/logger"
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/env"
-	"github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers"
 	"github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/eventtype"
+	nats3 "github.com/kyma-project/kyma/components/eventing-controller/pkg/handlers/nats"
```

Comment:
(nit) can we use better name than `nats3`?

Reply:
done